src/templates/finops-hub/modules/hub.bicep

```diff
@@ -137,6 +156,10 @@ module storage 'storage.bicep' = {
     scopesToMonitor: scopesToMonitor
     msexportRetentionInDays: exportRetentionInDays
     ingestionRetentionInMonths: ingestionRetentionInMonths
+    virtualNetworkId: vnet.outputs.vNetId
+    privateEndpointSubnetId: vnet.outputs.finopsHubSubnetId
+    scriptSubnetId: vnet.outputs.scriptSubnetId
+    enablePublicAccess: enablePublicAccess
```

Comment:
They're independent.
All our data processing happens within that subnet regardless of this setting.  
This just toggles the storage firewall on the data lake so that one can access it directly via the PBI service rather than deploying a gateway.